docs/source/federated-types/federated-directives.mdx

```diff
@@ -894,3 +894,46 @@ If different subgraphs use different versions of a directive's corresponding spe
 
 </tbody>
 </table>
+
+## Sharing data with contexts
```

Comment:
possibly copyedit, to reflect more saving data to reference elsewhere in type hierarchy